assets/js/mapper.js

```diff
@@ -145,29 +146,15 @@ class Mapper {
         slopeLabelGroup[3].appendChild(slopeLabelSilly);
         slopeLabelGroup[4].appendChild(slopeLabelSilly);
 
-
-        maxSlopeLabelGroup.setAttribute("class", "maxSlopeLabel");
-
         const drawOrder = {}; // [type, stroke-width, stroke]
-        if (this.config.ironOverWood) {
-            drawOrder[1] = [1, 15, 'darkkhaki']; // variable bank
-            drawOrder[2] = [2, 15, 'darkkhaki']; // constant bank
-            drawOrder[5] = [5, 15, 'darkgrey'];  // variable wall
-            drawOrder[6] = [6, 15, 'darkgrey'];  // constant wall
-            drawOrder[3] = [3, 15, 'orange'];    // wooden bridge
-            drawOrder[7] = [7, 15, 'lightblue']; // iron bridge
-            drawOrder[4] = [4, 3, 'black'];      // trendle track
-            drawOrder[0] = [0, 3, 'black'];      // track        darkkhaki, darkgrey, orange, blue, black
-        } else {
-            drawOrder[1] = [1, 15, 'darkkhaki']; // variable bank
-            drawOrder[2] = [2, 15, 'darkkhaki']; // constant bank
-            drawOrder[5] = [5, 15, 'darkgrey'];  // variable wall
-            drawOrder[6] = [6, 15, 'darkgrey'];  // constant wall
-            drawOrder[7] = [7, 15, 'lightblue']; // iron bridge
-            drawOrder[3] = [3, 15, 'orange'];    // wooden bridge
-            drawOrder[4] = [4, 3, 'black'];      // trendle track
-            drawOrder[0] = [0, 3, 'black'];      // track        darkkhaki, darkgrey, orange, blue, black
-        }
+        drawOrder[1] = [1, 15, 'darkkhaki']; // variable bank
+        drawOrder[2] = [2, 15, 'darkkhaki']; // constant bank
+        drawOrder[5] = [5, 15, 'darkgrey'];  // variable wall
+        drawOrder[6] = [6, 15, 'darkgrey'];  // constant wall
+        drawOrder[3] = [3, 15, 'orange'];    // wooden bridge
+        drawOrder[7] = [7, 15, 'lightblue']; // iron bridge
+        drawOrder[4] = [4, 3, 'black'];      // trendle track
+        drawOrder[0] = [0, 3, 'black'];      // track        darkkhaki, darkgrey, orange, blue, black

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ian76g/RailroadOnlineMapper/pullRequest?prid=8476050)
```suggestion
            drawOrder[0] = [0, 3, "black"];      // track        darkkhaki, darkgrey, orange, blue, black
```